alot/commands/envelope.py

```diff
@@ -168,6 +168,33 @@ def __init__(self, mail=None, envelope=None, **kwargs):
         self.envelope = envelope
         self.envelope_buffer = None
 
+    def _check_recipient_keys(self):
+        # get all recipient addresses
+        recipients = set()
+        for header in ['To', 'Cc', 'Bcc']:
+            values = self.envelope.headers.get(header)
+            if not values:
+                continue
+            for value in values:
```

Comment:
why not simply `for value in self.envelope.headers.get(header)` ? (I think you don't need the two lines above).